reference/redirects.v3.yml

```
@@ -2,7 +2,7 @@ openapi: 3.0.1
 info:
   title: Redirects
   description: |-
-    Manage 301 redirects for one or more storefronts powered by a single BigCommerce back-end.
+    Manage 301 redirects for one or more storefronts powered by a single BigCommerce back-end. For a list of redirects that are not allowed as an `/old-url/` parameter, see the 301 Redirects [FAQ](https://support.bigcommerce.com/s/article/301-Redirects#faq).
```

```suggestion
    Manage 301 redirects for one or more storefronts powered by a single BigCommerce backend. For a list of redirects that are not allowed, see the [301 Redirects FAQ](https://support.bigcommerce.com/s/article/301-Redirects#faq) in our Help Center.
```
- `backend`, per style guide
- The `/old-url/` thing seems confusing to me
- descriptive link text
- indicate that this link takes you away from DevDocs and into the Help Center.